agent/ec2/mocks/ec2_mocks.go

```diff
@@ -21,7 +21,9 @@ import (
 	reflect "reflect"
 
 	ec2 "github.com/aws/amazon-ecs-agent/agent/ec2"
+	ecs "github.com/aws/amazon-ecs-agent/agent/ecs_client/model/ecs"
 	ec2metadata "github.com/aws/aws-sdk-go/aws/ec2metadata"
+	ec20 "github.com/aws/aws-sdk-go/service/ec2"
```

Comment:
This ec20 is a little bit weird as a name. Maybe change '/agent/ec2' to **agent_ec2** and keep '/service/ec2' as **ec2**.

Reply:
I don't know, this is something auto-generated...

Reply:
its the worst!